pkg/gcs/sysfunc.go

```diff
@@ -261,6 +263,7 @@ func (e *Eval) setTargetPos(c *ast.CallExpr, env *Env) (Obj, error) {
 	}
 
 	e.Core.Combat.SetEnemyPos(idx-1, x, y)
+	e.Core.Combat.Player().SetDirectionToClosestEnemy()
```

Comment:
We might want to make it clear in documentation that moving enemies or the player will reset the target logic

Reply:
Sure, but updating documentation alongside every PR is the least of my concerns until it gets moved into this repo.